conmon-rs/server/src/console.rs

```diff
@@ -274,7 +274,7 @@ mod tests {
 
         // Write to the slave
         let mut file = unsafe { fs::File::from_raw_fd(res.slave) };
-        let _ = file.write(b"test").await?;
+        file.write(b"test").await?;
```

Comment:
This one is more subtle.  It should actually be `write_all()`.  While it doesn't usually happen for local files in the normal path, one almost always wants to use `write_all` unless dealing with a network socket.

Reply:
Yep, switched to `write_all`.